build/update-locales.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
```

Comment:
```suggestion
#!/bin/sh

# this script runs in alpine image and only has `sh` shell
```

Sorry, I made a mistake and I think it's incorrect to use bash in this script.

The last run fails: https://drone.gitea.io/go-gitea/gitea/68658/1/3

So we still need to use `sh`